products/zookeeper.md

```diff
@@ -33,9 +33,9 @@ releases:
 -   releaseCycle: "3.6"
     eol: false
     support: 2022-03-07
-    latest: "3.6.3"
+    latest: "3.6.4"
     releaseDate: 2020-02-25
-    latestReleaseDate: 2021-04-08
+    latestReleaseDate: 2022-12-18
```

Comment:
Auto conf could be switched to Maven : https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper/3.6.4.

Reply:
The API returns 18th December: https://search.maven.org/solrsearch/select?q=g:org.apache.zookeeper+AND+a:zookeeper&core=gav&rows=1&wt=json&start=%s

epoch: 1671387029

Reply:
The files are also dated 18th December: https://repo1.maven.org/maven2/org/apache/zookeeper/zookeeper/3.6.4/

(We're already using Maven for Zookeeper)

Reply:
> We're already using Maven for Zookeeper

Should have checked ! ;)

Strange, Dec 31, 2022 is displayed on https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper/3.6.4 (hence my comment). And https://www.mail-archive.com/dev@zookeeper.apache.org/msg75711.html also display Dec 31, 2022.

Reply:
Dropped a mail to info@mvnrepository.com